lookup/simple_env_test.go

```diff
@@ -0,0 +1,24 @@
+package lookup
+
+import (
+	"testing"
+)
+
+func TestOsEnvLookup(t *testing.T) {
+	osEnvLookup := &OsEnvLookup{}
+
+	envs := osEnvLookup.Lookup("PATH")
+	if len(envs) != 1 {
```

Comment:
Probably should also test with a setup environment variables a test the content though.